src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -3070,6 +3090,10 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
         }
     }
 
+    void visit_Instantiate(const AST::Instantiate_t &x){
```

Comment:
This will fix the warning:
```suggestion
    void visit_Instantiate(const AST::Instantiate_t &/*x*/){
```